README.md

```diff
@@ -1,3 +1 @@
-# DataDog UI Extensions SDK
-
-This packages provides a javascript SDK for DataDog UI Extensions. It is under active development and is subject to breaking changes at any time.
+# Datadog Apps
```

Comment:
This file will be replaced by whatever's in the `apps` repo's readme once we pull that in.